tests/fixtures/bar-foos-bar-route.js

```diff
@@ -0,0 +1,6 @@
+Router.map(function() {
```

Comment:
@nathanhammond I wouldn't expect nested redundant routes to be removed. Would you? If so, why? That seems a new feature.